containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzConsts.java

```diff
@@ -80,4 +80,7 @@
     public static final String ATHENZ_PROP_KEYSTORE_PASSWORD_APPNAME   = "athenz.ssl_key_store_password_appname";
     public static final String ATHENZ_PROP_KEYMANAGER_PASSWORD_APPNAME = "athenz.ssl_key_manager_password_appname";
     public static final String ATHENZ_PROP_TRUSTSTORE_PASSWORD_APPNAME = "athenz.ssl_trust_store_password_appname";
+
+    public static final String ATHENZ_PROP_GRACEFUL_SHUTDOWN_SUPPORT = "athenz.graceful_shutdown_support";
```

Comment:
```suggestion
    public static final String ATHENZ_PROP_GRACEFUL_SHUTDOWN = "athenz.graceful_shutdown";
```
Just graceful shutdown seems simpler. How do you think?

Reply:
Updated.